app/viewmodels/journeys/taskList/NationalInsuranceContributionsViewModel.scala

```diff
@@ -26,22 +28,28 @@ import viewmodels.journeys.{SummaryListCYA, getJourneyStatus}
 
 object NationalInsuranceContributionsViewModel {
 
-  def buildSummaryList(nationalInsuranceStatuses: List[JourneyNameAndStatus])(implicit messages: Messages): SummaryList = {
+  def buildSummaryList(
+      nationalInsuranceStatuses: List[JourneyNameAndStatus])(implicit messages: Messages, taxYear: TaxYear, businessId: BusinessId): SummaryList = {
 
-    val nicRow = buildRow(NationalInsuranceContributions, nationalInsuranceStatuses, dependentJourneyIsFinishedForClickableLink = false)
+    val nicRow = buildRow(NationalInsuranceContributions, nationalInsuranceStatuses, dependentJourneyIsFinishedForClickableLink = false)(
```

Comment:
dependentJourneyIsFinishedForClickableLink will be dependent on all the previous statuses being 'Completed'

Reply:
Done :)